rclcpp/test/resources/test_node_parameters/wildcards.yaml

```diff
@@ -0,0 +1,36 @@
+/**:
+  ros__parameters:
+    full_wild: "full_wild" 
+
+/*:
```

Comment:
I think we should have another test case for matching namespaces with multiple tokens. Following the design doc:

- `/*` should match exactly one token (e.g. `/foo` or `/bar`, but not `/foo/bar` or `/`)
- `/**` should match zero or more tokens (e.g. `/foo` or `/bar` or `/foo/bar` or `/`)